README.md

```diff
@@ -6,6 +6,7 @@ Warning : This script only works with ***Python3***
 
 ## Simple usage
 
+	>>> pip install -r requirements.txt
```

Comment:
I'm introducing a `requirements.txt` file for a dependency I needed.  Not sure how you'd feel about that.  If necessary, I could also figure out a way to do it without this new dependency (if nothing else, just lifting the code from the dependency).